circle.yml

```diff
@@ -55,6 +55,14 @@ executors:
       PLATFORM: mac
 
 commands:
+  install-latest-chrome:
+    steps:
+      - run:
+          command: |
+            apt update ;
```

Comment:
`apt-get` is usually recommended for scripts, but this is probably fine too since we aren't depending on the output